windows.sh

```diff
@@ -23,6 +23,7 @@ printf "\033[0m\n\n"
 
 # Check that add-apt-repository is installed for non-standard Vagrant boxes
 if [ ! -f /usr/bin/add-apt-repository ]; then
+  sudo apt-get update
   echo "Adding add-apt-repository..."
   sudo apt-get -y install software-properties-common
 fi
```

Comment:
There's another `sudo apt-get -y update` in the following conditional that installs Ansible. I suspect that both conditionals would typically run on the same occasion.

What do you think about combining the two conditionals with something like this?
```
if [ ! -f /usr/bin/add-apt-repository ] || [ ! -f /usr/bin/ansible ]; then
```
Then you could have the `sudo apt-get -y update` just once at the top of the combined conditional.